litmus-portal/authentication/api/handlers/user_handlers.go

```diff
@@ -46,7 +46,9 @@ func CreateUser(service user.Service) gin.HandlerFunc {
 }
 func UpdateUser(service user.Service) gin.HandlerFunc {
 	return func(c *gin.Context) {
+		uid := c.MustGet("uid").(string)
 		var userRequest entities.User
+		userRequest.ID = uid
```

Comment:
I think the assignment can be done after binding the request body, incase if anything goes wrong.